mqttclients/dotnet/MQTTnet.Client.Extensions/CommandClient.cs

```diff
@@ -50,23 +52,22 @@ public CommandConsumer(IMqttClient mqttClient, string cmdName, IMessageSerialize
             await Task.Yield();
         };
     }
-    public Task<TResp> InvokeAsync(string clientId, T request, int timeoutInSeconds = 5, CancellationToken ct = default)
+    public async Task<TResp> InvokeAsync(string clientId, T request, int timeoutInSeconds = 5, CancellationToken ct = default)
```

Comment:
Should be milliseconds everywhere.

Reply:
TimeSpan cannot be used as a default value. Updated to `ms`.